internal/util/commonpbutil/commonpbutil.go

```diff
@@ -57,20 +55,21 @@ func GetNowTimestamp() uint64 {
 	return uint64(time.Now().Unix())
 }
 
-func SetTargetID(msgBase *commonpb.MsgBase, targetID int64) *commonpb.MsgBase {
+func UpdateMsgBase(msgBase *commonpb.MsgBase, targetID int64) *commonpb.MsgBase {
```

Comment:
1. Rename to FillXXX is better
2. change the param from targetID to MsgBasOptions is better

Reply:
maybe combine this one with NewMsgBaseCopy is better 🤔